<launch>

    <arg name="publish_delay" default="1" />

    <node name="carla_helper" pkg="lidar_slam" type="carla_helper" output="screen">
        <param name="dataset_folder"    type="string"   value="/mnt/hdd2/Carla/" />
        <param name="town_number"       type="string"   value="Town04/" />
        <param name="sequence_number"   type="string"   value="19/" />
        <param name="velodyne_topic"    type="string"   value="/velodyne_points" />
        <param name="pose_topic"    type="string"   value="/gt_pose" />
        <param name="to_bag"            type="bool"     value="false" />
        <param name="output_bag_file"   type="string"   value="carla-Town04-19.bag" /> <!-- replace with your output folder -->
        <param name="publish_delay"     type="int"      value="$(arg publish_delay)" />
    </node>
</launch>